shotover-proxy/Cargo.toml

```diff
@@ -52,7 +52,7 @@ halfbrown = "0.1.11"
 
 # Transform dependencies
 redis-protocol = "3.0.1"
-cassandra-proto = { git = "https://github.com/shotover/cassandra-proto", branch = "move-to-bytes", features = ["v4"] }
+cassandra-proto = { git = "https://github.com/shotover/cassandra-proto", branch = "OPTION_fix", features = ["v4"]}
```

Comment:
I was able to get the test passing with https://github.com/shotover/cassandra-proto/pull/5 which only has the bare minimum of changes required.

![image](https://user-images.githubusercontent.com/5120858/141243480-96252347-6ef1-4db4-b84a-57b236c78e56.png)


cassandra-proto needs a lot of work but that should be done on the upstream repo, (or possibly a properly maintained fork if thats what it comes too)

But we cant just keep adding more changes onto bens poorly maintained fork.
So lets make the absolute minimum change to fix the version panic and then make the other cassandra-proto fixes once we land this shotover PR.

Reply:
I'm not sure what "that change" refers to:

* the screenshot
* cassandra-proto/move-to-bytes-version-fix
* cassandra-proto/OPTION_fix

?

Reply:
https://github.com/shotover/cassandra-proto/pull/5 has landed so you can remove this line and apply the other changes in the screenshot.

Reply:
hmmm maybe I can just close my eyes and ignore the large change to https://github.com/shotover/cassandra-proto

We do plan to move to cdrs-tokio's protocol implementation (cassandra-protocol) so its probably not worth thinking too hard about until we move to cassandra-protocol.